tests/test_tiny_rwkv.c

```diff
@@ -33,7 +33,7 @@ void test_model(const char * model_path, const float * expected_logits, const fl
     ASSERT(rwkv_gpu_offload_layers(model, N_GPU_LAYERS), "Unexpected error %d", rwkv_get_last_error(model));
 #endif
 
-    uint32_t n_vocab = rwkv_get_logits_len(model);
+    size_t n_vocab = rwkv_get_logits_len(model);
```

Comment:
`const size_t` I suppose

Reply:
I'll add it, but what will this accomplish? :)

Reply:
It makes the variable immutable